client/ayon_substancepainter/plugins/publish/collect_textureset_images.py

```diff
@@ -155,6 +155,14 @@ def create_image_instance(self, instance, template, outputs,
         image_instance.data["families"] = [product_type, "textures"]
         if instance.data["creator_attributes"].get("review"):
             image_instance.data["families"].append("review")
+            image_instance.data["fps"] = instance.context.data.get("fps", 25.0)
+            if bool(outputs[0].get("udim")):
+                udim = sorted(int(output["udim"]) for output in outputs)
+                image_instance.data["frameStart"] = udim[0]
+                image_instance.data["frameEnd"] = udim[-1]
+            else:
+                image_instance.data["frameStart"] = 1
+                image_instance.data["frameEnd"] = 1
```

Comment:
I wonder if 1001 may be a better 'default' here just for the sake of that being the start of UDIMs too?
```suggestion
                # Use start of UDIM range as fallback frame for single images
                image_instance.data["frameStart"] = 1001
                image_instance.data["frameEnd"] = 1001
```